woocommerce/woocommerce-ios
# Storage: Add new attribute `isAIAssitantFeatureActive` to `Site` entity.

⚠️ Please review https://github.com/woocommerce/woocommerce-ios/pull/10694 before this

Part of #10689  

## Description
This PR updates the Site entity in Storage layer with the new attribute `isAIAssitantFeatureActive`.

## Testing instructions
- Build and run the app after initially running it from trunk - the app should still run and should not crash. 
- CI should pass.

## Screenshots
N/A

---
